src/common/SPELLS/thewarwithin/food.ts

```diff
@@ -0,0 +1,367 @@
+import Spell from '../Spell';
+
+const food = {
+  /**
+   * 30 Fishing Skill, 150 Perception
+   */
+  GHOULFISH_DELIGHT: {
+    id: 456406,
+    name: 'Ghoulfish Delight',
+    icon: 'inv_cooking_90_cinnamonbonefishstew',
+  },
+  /**
+   * Supposed to give Lowest Secondary Stat and 1% size
+   *
+   * But right now resotres 0 mana per second until you stand up.
+   */
+  PROTEIN_SLURP: {
+    id: 456576,
+    name: 'Protein Slurp',
+    icon: 'inv_cooking_90_pickledmeatsmoothie_color03',
+  },
+  /** 582 Speed */
+  EXQUISITELY_EVISERATED_MUSCLE: {
+    id: 455369,
+    name: 'Exquisitely Eviscerated Muscle',
+    icon: 'inv_misc_food_84_roastclefthoof',
+  },
+
+  /**
+   * 469 Lowest Secondary Stat
+   */
+  WELL_FED_EVERYTHING_STEW: {
+    id: 461958,
+    name: 'Everything Stew',
+    icon: 'inv_cooking_10_draconicdelicacies',
+  },
+  /**
+   * 469 Lowest Secondary Stat
+   */
+  HEARTY_WELL_FED_EVERYTHING_STEW: {
+    id: 462182,
+    name: 'Hearty Everything Stew',
+    icon: 'inv_cooking_10_draconicdelicacies',
+  },
+  /**
+   * 469 Highest Secondary Stat
+   *
+   * - The Sushi Special
+   * - Beledar's Bounty
+   * - Jester's Board
+   * - Empress' Farewell
+   * - Outsider's Provisions
+   */
+  WELL_FED_HIGHEST_SECONDARY: {
+    id: 462181,
+    name: 'Hearty Well Fed',
+    icon: 'spell_misc_food',
+  },
+  /**
+   * 469 Highest Secondary Stat
+   *
+   * - Hearty Sushi Special
+   * - Hearty Beledar's Bounty
+   * - Hearty Jester's Board
+   * - Hearty Empress' Farewell
+   * - Hearty Outsider's Provisions
+   */
+  HEARTY_WELL_FED_HIGHEST_SECONDARY: {
+    id: 462183,
```

Comment:
The actual ID of the highest secondary changes based on the buff - you can use this report to check for the Haste one for example: /report/qfLWYgGAaF6xKzb4/11-Normal+Nexus-Princess+Ky'veza+-+Kill+(2:19)/Magelie/standard 

Tested it locally and it said I did not have a food buff.

Reply:
oh god please no 😢 

So by that logic, the 8 secondary stats buffs i manually tested, there should actually be 8x4 = 32 buffs???

Reply:
Ok so food that gives highest/lowest gives the same IDs at least. So updated (091cc151a511e31f5a99f3f08aed9593c4336d75) to correctly track all stats for 

 - (Hearty) Sushi Special
 - (Hearty) Beledar's Bounty
 - (Hearty) Jester's Board
 - (Hearty) Empress' Farewell
 - (Hearty) Outsider's Provisions
 - (Hearty) Everything Stew
 
 Will have to add IDs for all stats for the Quick and Easy and Snacks tomorrow 😰

Reply:
I went trawling through the IDs, way too difficult to manually test all buffs, but they're all there now! Thanks @avilene for catching that, but wow that sucks haha